app/src/main/AndroidManifest.xml

```diff
@@ -14,12 +14,18 @@
         </activity>
         <activity
             android:name=".ChooseMemeStyle"
-            android:label="@string/title_activity_choose_meme_style">
-        <intent-filter>
-            <action android:name="android.intent.action.MAIN" />
+            android:label="@string/title_activity_choose_meme_style" >
```

Comment:
before this is merged, remember to change it back so that MainActivity is the launcher activity.